packages/embed/src/factories/create-widget/create-widget.ts

```diff
@@ -8,6 +9,13 @@ export type Widget = {
   unmount: () => void
 }
 
+const buildCloseButton = () => {
```

Comment:
I dont think it is necessary to test the build method itself, especially since there is no logic here. 

Testing just the `createWidget` should be sufficient and writing a test there will also allow us to test the "close" functionality, which we can not test here.